packages/forms/src/lib/search-filter/README.md

```diff
@@ -0,0 +1,83 @@
+# @acpaas-ui/ngx-components/forms
+
+The search-filter is a JIRA-like filter component with remote search capabilities.
+
+## Usage
+
+```typescript
+import { SearchFilterModule } from '@acpaas-ui/ngx-components/search-filter'`;
+```
+
+## Documentation
+
+Visit our [documentation site](https://acpaas-ui.digipolis.be/) for full how-to docs and guidelines
+
+### API
+
+| Name         | Default value | Description |
+| -----------  | ------ | -------------------------- |
+| `@Input() id: string;` | - | Field id. |
+| `@Input() name: string;` | - | Field name. |
+| `@Input() flyoutSize: enum;` | `FlyoutSize.Small` | Passed along to `auiFlyout`. |
```

Comment:
Perhaps describe it a bit more. It's straight forward, but you can copy some content from the flyout.

Reply:
Suggestion: maybe it's better to provide link to the appropriate documentation?